src/Backend/Version.ts

```diff
@@ -23,7 +23,7 @@ class Version {
   constructor(
     major: number,
     minor: number | undefined,
-    patch: number | undefined,
+    patch?: number | undefined,
```

Comment:
You can use `undefined` value instead of changing this line.

Reply:
Thank you for your advice~!! I modified EdgeTPU Compiler Version to use undefined patch instead of changing the Version class

https://github.com/SSDC-onE-FiVE/ONE-vscode/pull/71